package.json

```diff
@@ -51,7 +51,7 @@
     "@typescript-eslint/parser": "^5.33.0",
     "@zowe/cli": "^7.9.7",
     "@zowe/cli-test-utils": "^7.9.7",
-    "@zowe/imperative": "^5.8.0",
+    "@zowe/imperative": "^5.8.1",
```

Comment:
Isn't imperative now up to 5.8.2?

Reply:
Yes, the Imperative version is updated to 5.8.2 in npm-shrinkwrap, but the minimum version required to update the snapshots was 5.8.1.

Reply:
ok. Thanks for the clarification.